requirements.txt

```diff
@@ -6,6 +6,10 @@
 #
```

Comment:
We need to use pip-compile to update requirements_dev.txt also, as it copies from requirements.txt at that point, rather than including it at install time (as requirements_dev.*in*) does.